services/121-service/src/activities/interfaces/data-change-activity.interface.ts

```diff
@@ -0,0 +1,12 @@
+import { ActivityTypeEnum } from '@121-service/src/activities/enum/activity-type.enum';
+import { BaseActivity } from '@121-service/src/activities/interfaces/base-activity.interface';
+
+export interface DataChangeActivity extends BaseActivity {
+  type: ActivityTypeEnum.DataChange;
+  attributes: {
+    fieldName: string | null;
```

Comment:
What is the necessity of having the fields to be | null? Not having this would make using the interface simpler.

Reply:
Ok, I would like to discuss this with @PeterSmallenbroek , related to the ##TODOs already in the code.